.github/workflows/build.yml

```diff
@@ -183,6 +183,7 @@ jobs:
             -
                 name: Create-project with skeleton
                 run: |
+                    set -x
```

Comment:
I added back this debug mode and that works... strange behaviour here.
Maybe that's a luck here.